pkg/client/event.go

```diff
@@ -37,7 +37,7 @@ func initEvent(kubeClient kubernetes.Interface) {
 		},
 	)
 	eventRecorder = eventBroadcaster.NewRecorder(
-		Scheme, v1.EventSource{Component: "directcsi-controller"},
+		runtime.NewScheme(), v1.EventSource{Component: "directcsi-controller"},
```

Comment:
We don't need to use `runtime.NewScheme()` here, we can copy `pkg/utils/scheme.go` -> `pkg/client`